src/main/cli/cli.c

```diff
@@ -6121,120 +6220,169 @@ static void cliHelp(char *cmdline)
     }
 }
 
-void cliProcess(void)
+static void processCharacter(const char c)
 {
-    if (!cliWriter) {
+    if (c == '\t' || c == '?') {
+        // do tab completion
```

Comment:
should tab completion (and other special characters for humans) be specifically ignored when processing defaults?

Reply:
Good point, I will exclude these.